flight/Modules/RadioComBridge/RadioComBridge.c

```diff
@@ -104,6 +106,16 @@ static void ProcessRadioStream(UAVTalkConnection inConnectionHandle,
 
 static RadioComBridgeData *data;
 
+static void NewReceiverData(UAVObjEvent * ev, void *ctx, void *obj, int len)
+{
+	(void) ctx; (void) obj; (void) len;
```

Comment:
uncrustify will squash that space one day :stuck_out_tongue:

Reply:
:) I'm fine with any convention.  Linux kernel style doesn't care and seems to be about half and half whether there's a space after the cast.